app/config/AppConfig.scala

```diff
@@ -105,6 +105,8 @@ class AppConfig @Inject()(
   lazy val draftTimeToLive: FiniteDuration =
     servicesConfig.getDuration("draft.timeToLive").asInstanceOf[FiniteDuration]
 
+  lazy val draftItemPerPage: Int = servicesConfig.getInt("draft.itemsPerPage")
```

Comment:
I imagine the 50 limit will be service wide rather than just for drafts

Reply:
But no harm keeping it as it is :)

Reply:
You're probably right.  I'll rename it.